helm_deploy/create-and-vary-a-licence/templates/activate-licences-cronjob.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: activate-licences
+spec:
+  schedule: "0 19 * * *"
+  concurrencyPolicy: Replace
```

Comment:
I would suggest an early morning run time for this, or maybe even twice per day - early morning + after lunchtime?

Reply:
Can do, this time now is temporary, I will have a follow-up PR to set the time to early morning once we can see it working

Reply:
7pm this evening has passed.. so I'll set it again when ready to merge

Reply:
Actually, looking at the code - it only checks against the date, not a date and time so a once-per-day run is enough.

Reply:
Unless there are licences which are only approved on the day they are supposed to be released!

Reply:
Aha. And there will be - so good shout - perhaps early evening to get today's releases.